ert/storage/_storage.py

```diff
@@ -85,17 +85,18 @@ async def _transmit_recordtree(
         storage_url = self._uri[: self._uri.rfind("/")]
         for record_path in record.flat_record_dict:
             record_name = record_path.split("/")[-1]
-            transmitter = StorageRecordTransmitter(record_name, storage_url)
+            transmitter = StorageRecordTransmitter(
+                record_name, storage_url, iens=self._real_id
+            )
             await transmitter.transmit_record(record.flat_record_dict[record_path])
             data[record_path] = transmitter._uri
         await self._transmit_blob_record(
             ert.data.BlobRecord(data=json.dumps(data).encode("utf-8"))
         )
-        if self._real_id:
-            url = f"{self._uri}/userdata?realization_index=0"
-        else:
-            url = f"{self._uri}/userdata?"
-        await _put_to_server_async(url, {}, json={"record_type": record._record_type})
+        if self._real_id == 0:
+            await add_record_metadata(
```

Comment:
Should there be a comment here about why we treat realization 0 differentl? What about ensembles where realization 0 crashes, or an evaluation run where only a said interval of realizations are run?

Reply:
Yes, definitely.

Reply:
This'll change in the near future (see https://github.com/equinor/ert-storage/issues/195). Wrt. record metadata the current consensus is it to annotate `real_id==0` only (see `add_record_metadata`).

Reply:
Added comment ...